react-facet/docs/api/hooks.md

```diff
@@ -22,13 +22,25 @@ Returns a `[facet, setFacet]` pair. Like React’s `useState`, but with a Facet
 
 This example illustrates how to use this hook in the common use case of having to store the temporary state of the input field until it is submitted.
 
-```tsx
-const Form = ({ onSubmit, initialValue }) => {
+```tsx twoslash
+// @esModuleInterop
+// @errors: 7006
```

Comment:
I needed to add this error suppression [because](https://github.com/Mojang/ore-ui/runs/6173849367?check_suite_focus=true#step:4:109). Not sure why